core/CHANGELOG.md

```diff
@@ -2,3 +2,20 @@
 
 * Renamed ItemAdjustments to Adjustable::AdjustmentsUpdater.
 * Removed allow_ssl_in_* preferences from spree
+* Enable configuration of `Carmen.i18n_backend.locale` at `seeds.rb`
```

Comment:
It would actually be better to place this changelog entry within the release notes guide instead.  I think the changelog files should be removed as it's better for us to simply add to release notes rather than adding to changelog then moving to release notes later.